packages/release-brancher/src/release-brancher.ts

```diff
@@ -330,7 +336,7 @@ export class Runner {
         }
       }
     }
-    const message = 'feat: configure initial sp version';
+    const message = 'feat: configure the protected branch';
```

Comment:
@chingor13 Does "configure initial sp version" have special effect somewhere? In past (https://github.com/googleapis/repo-automation-bots/pull/1674/files#diff-4b2e8d3f62c327048db13c5f552d94303eac7ed241728c5fd5dbc3e5842961c9R327), you updated this message from 'ci: setup workflows for new release branch' to 'feat: configure initial sp version'.

Reply:
In this case, any releasable commit (`feat`, `fix`, etc, will trigger a new library release). For the -sp versions, we wanted to force a release. For the new backport releases, we don't need to force a new release, so `ci` is probably a better message.

Reply:
Created https://github.com/googleapis/repo-automation-bots/pull/2622#pullrequestreview-766165606